docs/source/user/10minute_intro.md

```diff
@@ -1,7 +1,246 @@
+```
+
+
+### 1.3 Example: Downloading a catalog from a FDSN-Server
+Since downloads are limited to a maximum of 20,000 events, you can use the batch download option for larger datasets. This method downloads the data in subsets of events based on the specified `batch_size`.
+```python
+>>> from seismostats.catalogs.client import FDSNWSEventClient
+>>> from seismostats import Catalog
+>>> import pandas as pd
+
+>>> # Define time range and region of interest
+>>> start_time = pd.to_datetime('2020/01/01')
+>>> end_time = pd.to_datetime('2022/01/01')
+
+>>> min_longitude = 5
+>>> max_longitude = 11
+>>> min_latitude = 45
+>>> max_latitude = 48
+
+>>> min_magnitude = 0.5
+>>> url = 'http://eida.ethz.ch/fdsnws/event/1/query'
+>>> client = FDSNWSEventClient(url)
+
+>>> # Download events
+>>> df = client.get_events(
+...     start_time=start_time,
+...     end_time=end_time,
+...     min_magnitude=min_magnitude,
+...     min_longitude=min_longitude,
+...     max_longitude=max_longitude,
+...     min_latitude=min_latitude,
+...     max_latitude=max_latitude,
+...     batch_size=1000)
+
+>>> # Create catalog and preview entries
+>>> cat = Catalog(df)
+>>> cat.head()
+  event_type             time                  latitude  longitude      depth evaluationmode  magnitude magnitude_type  magnitude_MLhc  magnitude_MLh
+0  earthquake  2021-12-30 07:43:14.681975   46.051445   7.388025  1181.640625        manual   2.510115           MLhc     2.510115344            NaN
+1  earthquake  2021-12-30 01:35:37.014056   46.778985   9.476219  9294.921875        manual   1.352086           MLhc     1.352086067            NaN
+2  earthquake  2021-12-29 08:48:59.059653   47.779511   7.722354 16307.812500        manual   0.817480           MLhc     0.8174796651           NaN
+
+```
+
+---
+
+## 2 Visualizing the Catalog
+
+The `Catalog` class offers several built-in methods for analyzing and visualizing your seismic data.
+
+These tools allow you to:
+
+- Quickly explore spatial and temporal patterns
+- Inspect magnitude distributions
+- Generate publication-ready plots with minimal code
+
+Use these methods to gain insights into your catalog before performing more advanced statistical analyses.
+
+```python
+>>> # import matplotlib for creating a combined plot
+>>> import matplotlib.pyplot as plt
+
+>>> # plot basic information of the catalog
+>>> fig, ax = plt.subplots(3, 1, figsize=(8, 10))
+>>> ax[0].set_title('Catalog Cumulative Count')
+>>> cat.plot_cum_count(ax=ax[0],mcs=[1,2])
+>>> ax[1].set_title('Magnitudes in Time')
+>>> cat.plot_mags_in_time(ax=ax[1])
+>>> ax[2].set_title('Frequency Magnitude Distribution')
+>>> cat.plot_cum_fmd(ax=ax[2])
+>>> cat.plot_fmd(ax=ax[2])
+>>> plt.tight_layout()
+>>> plt.show()
+
+>>> # Earthquake map:
+>>> fig = plt.figure(figsize=(8, 5))
+>>> cat["latitude"]= cat["latitude"]
+>>> cat.plot_in_space(include_map=True)
+>>> plt.show()
+```
```

Comment:
maybe split this up into 2-3 paragraphs and explain what you're doing - there is a concept behind this and if you're not very familiar with matplotlib you probably won't get it.